static/app/views/performance/transactionSummary/transactionVitals/utils.tsx

```diff
@@ -155,3 +157,27 @@ export function mapPoint(
     y: destRect.point1.y + (destRect.point2.y - destRect.point1.y) * yPercentage,
   };
 }
+
+export function shouldDisplayMissingVitalsAlert(
+  vitalsData: VitalsData | null,
+  allVitals: WebVital[]
+): boolean {
+  if (!vitalsData) {
+    return true;
+  }
+
+  const isMissingVitalsMeasurement = allVitals.some(vital => !vitalsData[vital]);
+
+  const measurementsWithoutData = Object.keys(vitalsData).filter(key => {
+    const vitalObj = vitalsData[key];
+    const isVitalObjectEmpty = Object.keys(vitalObj).every(
+      measurement => vitalObj[measurement] === null || vitalObj[measurement] === 0
+    );
+
+    return isVitalObjectEmpty;
+  });
+
+  const hasEmptyVitalsMeasurement = measurementsWithoutData.length > 0;
+
+  return isMissingVitalsMeasurement || hasEmptyVitalsMeasurement;
+}
```

Comment:
Just checking the totals is simpler.
```suggestion
export function shouldDisplayMissingVitalsAlert(
  vitalsData: VitalsData | null,
  allVitals: WebVital[]
): boolean {
  if (!vitalsData || allVitals.some(vital => !vitalsData[vital])) {
    return true;
  }
  
  const measurementsWithoutCounts = Object.values(vitalsData).filter(vitalObj => vitalObj.total > 0);
  return measurementsWithoutCounts.length > 0;
}
```

Reply:
I thought about this but was afraid the check wouldn't be rigorous enough lol.
But that's probably just me being too paranoid, and I think this check is good enough and makes the code a lot more readable 👍